src/styles/card.css

```diff
@@ -0,0 +1,53 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  background: #fff;
+  color: #060b35;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto 40px;
+}
+
+.title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
```

Comment:
you've encountered a margin collapse with default h3 styles:
![default style margin collapse](https://user-images.githubusercontent.com/39308717/160771476-e9a0c129-69ea-4f4e-80c5-cc941e19ce10.png)
you need to reset margins before applying margin-bottom